unfoldingWord/translationCore
# Add timestamp generator helper

#### This pull request addresses:

This pull request brings in a helpers folder, with an index, and the first helper being the timestampGenerator. 


#### How to test this pull request:

Try importing the timestamp generator and using it to generate a timestamp. Also, make sure I'm doing the index correctly.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/942)
